### PR TITLE
Add JWT middleware and `me` query

### PR DESCRIPTION
**Pull request by KillerX**
The middleware is applied to all requests. If the `Authorization` header is not set the request is marked as Anonymous.
The middleware is not responsible for preventing access, it is only supposed to validate the token (if it exists) and extract claims into the context. Claims retrieved from the context later can thus be considered valid and used to Authorization purposes.

### CODE REVIEW

File: backend/graph/schema.graphqls [hunk 1]
```diff
@@ -247,6 +247,14 @@ type FAQCategory {
   questions: [FAQ!]!
 }
 
+type User {
+  personId: ID
```
Comment (fredrikved) [inline backend/graph/schema.graphqls:251]: Do we want this to be called personId? One could argue that this is a bad naming originally in Members...? 

I'd either go with "id" or "userId".

Comment (fredrikved) [reply backend/graph/schema.graphqls:251]: Of course if this is just a "forward" of what we get from auth0, then we don't have much of a choice

**Review by fredrikved — APPROVED** [review]

**Comment by andreasgan** [comment]
I'm curious if we need to check hasMembership.
I've never done that
https://github.com/bcc-code/bcc-code.github.io/blob/master/_docs/signon/openid-connect.md

File: backend/graph/schema.graphqls (hunk 1 above)
Comment (KillerX) [reply backend/graph/schema.graphqls:251]: Yeah, I'll move this to ID, makes more sense, even if members i "wrong".

**Comment by KillerX** [comment]
> I'm curious if we need to check hasMembership. I've never done that https://github.com/bcc-code/bcc-code.github.io/blob/master/_docs/signon/openid-connect.md

I will use this for https://github.com/bcc-code/brunstadtv/pull/21/files#diff-fcaa1411fa7893dd366fed7c9738f2dc0b054503ed682c3df089fbd7946155b0R22 thanks

**Pull request merged**